### PR TITLE
add release assistant

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- add release assistant for SDK

### CODE REVIEW

File: scripts/ping_slack_about_package_release.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+if [[ ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    curl -X POST -H "Content-type: application/json" \
+	 --data "{\"text\":\"💫 New *Rasa SDK* version ${TRAVIS_TAG} has been released! Changelog: https://github.com/RasaHQ/rasa-sdk/blob/${TRAVIS_TAG}/CHANGELOG.rst\"}" \
```
Comment (tmbo) [inline scripts/ping_slack_about_package_release.sh]: let's use a different emoji e.g.
```suggestion
	 --data "{\"text\":\"😻 New *Rasa SDK* version ${TRAVIS_TAG} has been released! Changelog: https://github.com/RasaHQ/rasa-sdk/blob/${TRAVIS_TAG}/CHANGELOG.rst\"}" \
```

File: .travis.yml [hunk 2]
```diff
@@ -40,3 +40,4 @@ jobs:
         tags: true
       password:
         secure: "a1ur/UMpxoPv6bTPC8Bgc9UVKG/FU8IZR2+dnr5oq3XGPP1WkyyEed95QrSQ+QNHgHPjSUMRxu1zuflQbxAW9i+y+1qLjKMHc1Xr1aUsuXIOjrTg62hKpB6Wx5hRBFPZXchU7mr6rVF7m+iQtK+dy0P7Qoffou603TzivYTDT1N9640gzlAWjeBjyC46KBjpXVJlIEaGNqiNCsbC7B0g3hCgdIIOmNGW+aoiZsy+LU6c/coADsly7CrHdld0GuGNxeHZDvVv6EJGkc8igLnmRsgoU9rlq/ZSlF3Ob8hXOzy20dm9XU+zTNcQglkyI6ebSOywrGtv4Rc6y+Ik5Qjg958Eujme7KephQ5l7hqkkp6kPMThhLHss7mH8hnxiFeTdMCrKZshektsSWNsYAV07cL9S52VOI47Nm7gpqXjtYhNu7MHbC/jio42k+wbZvVHNq71/S63ejosf1IO5pqlCajnSRHCaQH3mDrmuV9WyFgNL/Hdwp9do95XE+fyWAq3h/x6m7SxvzLbYQ+Km4fvlW5apb6XZLyFFXXaauxpLzPq5Cm/amrGx85+n9vH+PMtpu2mCmlgfCT3RV97AV9yNgGUvTMCVVXMV0n3EdX8tAGu7XkEbzDsxTAUSbJt3x+/d9gCn/FVStp02AcTlHgyFl+S6vta6Howp5nzY+KjZU0="
+    after_deploy: ./scripts/ping_slack_about_package_release.sh
```
Comment (tmbo) [inline .travis.yml]:
```suggestion
    after_deploy: ./scripts/ping_slack_about_package_release.sh

```

File: scripts/ping_slack_about_package_release.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+	 "https://hooks.slack.com/services/T0GHWFTS8/BMTQQL47K/${SLACK_WEBHOOK_TOKEN}"
```
Comment (tmbo) [inline scripts/ping_slack_about_package_release.sh]: The `SLACK_WEBHOOK_TOKEN ` needs to be added to travis envs

File: scripts/ping_slack_about_package_release.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (tmbo) [inline scripts/ping_slack_about_package_release.sh]: did you `chmod +x` this script?

File: scripts/ping_slack_about_package_release.sh (hunk 1 above)
Comment (erohmensing) [reply scripts/ping_slack_about_package_release.sh]: Haha it is different, the other use ⚡️ and 💥 . I like the cat though 😄

File: scripts/ping_slack_about_package_release.sh (hunk 3 above)
Comment (erohmensing) [reply scripts/ping_slack_about_package_release.sh]: I was wondering where those came from, I couldn't find them in either of the other repos, so thought it was just something in our travis setup somewhere. Where can i do that?

File: scripts/ping_slack_about_package_release.sh (hunk 4 above)
Comment (erohmensing) [reply scripts/ping_slack_about_package_release.sh]: did now

File: scripts/ping_slack_about_package_release.sh (hunk 3 above)
Comment (tmbo) [reply scripts/ping_slack_about_package_release.sh]: added it to travis :+1:

**Pull request merged**